python/utils.py

```diff
@@ -187,19 +188,43 @@ def sweep_ref(I, Rmin, dR, nrow, ncol, dxy, inc, Dx=0., Dy=0., dtype_image='floa
     """
     Compute the intensity map (i.e. the image) given the radial profile I(R)=ints.
     We assume an axisymmetric profile.
+    The output image is assumed with origin in the upper left corner.
```

Comment:
> output image is assumed

we produce the output image, so we don't assume. Let's say

The origin of the output image in the upper left corner.

Reply:
Agree. fixed